readthedocs/builds/managers.py

```diff
@@ -67,6 +72,12 @@ def create_latest(self, **kwargs):
         defaults.update(kwargs)
         return self.create(**defaults)
 
+    def get_object_or_log(self, **kwargs):
+        try:
+            return super(VersionManagerBase, self).get(**kwargs)
```

Comment:
We are running py3 only. We should replace this with just `super().get`

Reply:
Thank you.
I have pushed the changes.